intro-HPC/ch_troubleshooting.tex

```diff
@@ -0,0 +1,58 @@
+\chapter{Troubleshooting}
+\label{ch:troubleshooting}
+
+
+
+\section{Walltime issues}
+If you get from your job output error a message similar to this:
+
+\begin{prompt}
+ =>> PBS: job killed: walltime 146 exceeded limit 120
+\end{prompt}
+
+You job has exceeded the requested walltime. Probably the job requires more time to be executed.
```

Comment:
`This occurs when your job did not complete within the requested walltime (in this case, 86400 seconds which is equivalent to 24 hours).`